### PR TITLE
Gitlab: Support more event types

### PR DESCRIPTION
**Pull request by Loskir**
## Description
- Add support for "joined project"
- Add support for tags
- Add default item for unsupported events

Addresses #943 

## Screencast
изображение
изображение
изображение

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)

### CODE REVIEW

File: extensions/gitlab/src/components/event.tsx [hunk 1]
```diff
@@ -72,32 +84,41 @@ export function EventListItem(props: { event: Event }) {
...
-              case "deleted":
-                {
-                  icon = { source: GitLabIcons.branches, tintColor: Color.Red };
-                }
-                break;
-            }
+            iconSource = GitLabIcons.branches;
+          } else if (pd.ref_type === "tag") {
+            title = `${an} tag ${pd.ref}`;
+            iconSource = Icon.Pin; // todo: use gitlab tag icon
```
Comment (tonka3000) [inline extensions/gitlab/src/components/event.tsx:111]: Change the icon first?

Comment (Loskir) [reply extensions/gitlab/src/components/event.tsx:111]: Could you give some advice?
I've found this one (https://gitlab.com/gitlab-org/gitlab-svgs/blob/main/sprite_icons/tag.svg) but I couldn't download it as PNG and I also don't really like it. What do you think?

Comment (tonka3000) [reply extensions/gitlab/src/components/event.tsx:111]: https://svgtopng.com could be an option. Just make sure that the png is big enough.

Comment (tonka3000) [reply extensions/gitlab/src/components/event.tsx:111]: But I'm also fine with the Raycast icon.

**Comment by Loskir** [comment]
image
I like it

**Comment by tonka3000** [comment]
@pernielsentikaer @mattisssa I'm fine with the PR

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @Loskir 👋 

Thanks for your contribution, awesome! 

Did you consider adding yourself as [contributors in the package.json](https://developers.raycast.com/information/manifest#extension-properties)?

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Review by pernielsentikaer — APPROVED** [review]
Hi @Loskir 👋 

Thanks for your update 💪 

I'm going to approve it, let's await final review 👍

**Review by mathieudutour — APPROVED** [review]

**Pull request merged**